docs/release-notes/2.41.0.md

```diff
@@ -44,6 +44,11 @@ We have decided to separate our DBaaS offering into an independent product. Cons
 While the DBaaS functionality will remain available in PMM versions 2.x, all future updates and enhancements will be exclusively accessible through the Percona Everest interface.
 For a more streamlined and robust database deployment experience, try [Percona Everest](http://per.co.na/pmm-to-everest).
 
+### Technical Preview of PMM in HA mode
+!!! caution alert alert-warning "Important/Caution"
+    Disclaimer: This feature has been added in PMM 2.41.0 and is currently in [Technical Preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview). Early adopters are advised to use this feature for testing purposes only as it is subject to change.
+
```

Comment:
@rnovikovP Please check the description for this feature:

Starting with this release, PMM can now be run in high-availability mode with several PMM server applications and independent data sources.

Currently, PMM servers provide high availability, but users are responsible for maintaining the high availability of the data sources used by PMM. 

These data sources are:
ClickHouse: An open-source, fast analytical database.
VictoriaMetrics: A scalable, long-term storage solution for time series data.
PostgreSQL: A powerful open-source relational database management system used in this setup to store PMM data like inventory, settings, and other feature-related data.